src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -474,191 +474,82 @@ typedef struct
 
 struct PROBLEME_HEBDO
 {
+    /* Business problem */
     char OptimisationAuPasHebdomadaire;
-
-    char ReinitOptimisation;
-
-    char SecondeOptimisationRelaxee;
-
-    char MarketPoolActivated;
-    char FlexUpPoolActivated;
-    char FlexDownPoolActivated;
-
-    char ImpressionDuCritere;
-
-    char UnitCommitmentExact;
-
     char TypeDeLissageHydraulique;
-
-    char ExportMPS;
-    bool exportMPSOnError;
-    bool ExportStructure;
-
     char WaterValueAccurate; /* OUI_ANTARES /NON_ANTARES*/
-
     char OptimisationAvecCoutsDeDemarrage;
-    int NbTermesContraintesPourLesCoutsDeDemarrage;
-
-    char BorneDeLaDefaillancePositive;
-
-    char* DefaillanceNegativeUtiliserPMinThermique;
-
-    char* DefaillanceNegativeUtiliserHydro;
-
-    char* DefaillanceNegativeUtiliserConsoAbattue;
-
-    double* CoefficientEcretementPMaxHydraulique;
-
-    double** BruitSurCoutHydraulique;
-
-    unsigned int HeureDansLAnnee;
-
-    char LeProblemeADejaEteInstancie;
-    char TypeDOptimisation;
-
     int NombreDePays;
     const char** NomsDesPays;
     int NombreDePaliersThermiques;
 
-    int NumberOfMarketOfferPool;
```

Comment:
**Caution :**
a bunch of variables were introduced here as a start for a future work : down flexibility of thermal energy

Reply:
I've restored unused variables, with a comment and an `#if 0` condition surrounding their definition.